collections/_article/dev-snapshot-godot-4-2-beta-1.md

```diff
@@ -0,0 +1,346 @@
+---
+title: "Dev snapshot: Godot 4.2 beta 1"
+excerpt: ""
+categories: ["pre-release"]
+author: Yuri Sizov
+image: /storage/blog/covers/dev-snapshot-godot-4-2-beta-1.webp
+image_caption_title: Lush Jungle
+image_caption_description: A demo scene by Wrobot
+date: 2023-10-12 14:00:00
+---
+
+Over the course of [the last 3 months](/article/godot-4-1-is-here) **270 contributors** submitted more than **1200 improvements** for the Godot 4.2 release!
+
+Among these changes are both necessary bugfixes and exciting new features, turning Godot 4 into an even better and more polished tool for you to realize your game and app ideas. There are many more improvements to come, but for now we draw the line here and turn our efforts towards stabilizing the engine so it can be released officially at the start of November.
+
+This means, _it's beta testing time,_ and you are invited to offer your hand in squashing bugs and identifying usability issues. Help us make sure that in Godot 4.2 there are no crashes, unexpected regressions, and performance is no worse (but preferably better) than before. Godot is developed by its community, and it's thanks to people like you we can be sure in the quality of our collective work!
```

Comment:
```suggestion
This means, _it's beta testing time,_ and you are invited to offer your hand in squashing bugs and identifying usability issues. Help us make sure that in Godot 4.2 there are no crashes or unexpected regressions, and that performance is no worse than before. Godot is developed by its community, and it's thanks to people like you we can be sure in the quality of our collective work!
```
Before, it sounded like this was the start of a long list with each item starting with "no", i.e. "Make sure no crashes, no regressions, and no performance..." which suddenly jolts the reader when they realise the performance bit is not part of the "no" list. Also removed the bracketed text as it made the sentence a bit long and seemed unnecessary or distracting. (Gosh, it takes longer to write these explanations than just making the changes!)

Reply:
I prefer to keep the part in parentheses as a stylistic choice. I'll apply the rest though.

```suggestion
This means, _it's beta testing time,_ and you are invited to offer your hand in squashing bugs and identifying usability issues. Help us make sure that in Godot 4.2 there are no crashes or unexpected regressions, and that performance is no worse (but preferably better) than before. Godot is developed by its community, and it's thanks to people like you we can be sure in the quality of our collective work!
```